api-docs/rst/dev-guide/concepts.rst

```diff
@@ -714,7 +827,7 @@ monitoring service. Event-based configuration works the following way:
 
 #. The webhook launches the Autoscale service, which looks up the
    policy that has been defined in accordance with the webhook. This
-   policy determines the amount of cloud servers that need to be added
+   policy determines the amount of units that need to be added
    or removed.
```

Comment:
... determines the number or percentage of units to be added or removed from the scaling group.